SampleDanger/ViewController.swift

```diff
@@ -12,6 +12,645 @@ class ViewController: UIViewController {
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: SampleDanger/ViewController.swift;
  Line: 649;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)